tests/datasets-memcap-02/test.yaml

```diff
@@ -0,0 +1,14 @@
+pcap: ../flowbit-oring/input.pcap
+
+requires:
+  min-version: 8
+
+exit-code: 1
+
+args:
+ - -k none
+
+checks:
+    - shell:
+        args: grep "dataset too large for set memcap" suricata.log | wc -l
```

Comment:
How come this one line small dataset is too large for memcap 88074 ?

Reply:
Because datasets have a prealloc hardcoded setting of 1000. Size of the THashData object of String type is 88B. So, you already have 88000B alloc'd. and then, one hash row is 64B. So, you get total memuse of 88064.

Reply:
Oh, thanks